CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ else(AKTUALIZR_VERSION)
     add_definitions(-DAKTUALIZR_VERSION="dev")
 endif(AKTUALIZR_VERSION)
 
+if(PROVISION_FILE)
+    add_definitions(-DPROVISION_FILE="${PROVISION_FILE}")
+endif(PROVISION_FILE)
+
```

Comment:
A small issue, but can we change this name to `SOTA_PACKED_CREDENTIALS`, please?  That is the name which we use elsewhere (e.g. https://docs.atsgarage.com/quickstarts/qemuvirtualbox.html), and reusing the same name will make things a little bit easier for our users.